cmd/skupper/skupper_link.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"github.com/spf13/cobra"
+
+	"github.com/skupperproject/skupper/api/types"
+)
+
+func NewCmdLink() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "link create <input-token-file> [--name <name>] or link delete ...",
+		Short: "Manage skupper links definitions",
+	}
+	return cmd
+}
+
+var connectorCreateOpts types.ConnectorCreateOptions
+
+func NewCmdLinkCreate(newClient cobraFunc, flag string) *cobra.Command {
+
+	if flag == "" { //hack for backwards compatibility
+		flag = "name"
+	}
+
+	cmd := &cobra.Command{
+		Use:    "create <input-token-file>",
+		Short:  "Links this skupper installation to that which issued the specified connectionToken",
+		Args:   cobra.ExactArgs(1),
+		PreRun: newClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			silenceCobra(cmd)
+			siteConfig, err := cli.SiteConfigInspect(context.Background(), nil)
+			if err != nil {
+				fmt.Println("Unable to retrieve site config: ", err.Error())
+				os.Exit(1)
+			} else if siteConfig == nil || !siteConfig.Spec.SiteControlled {
+				connectorCreateOpts.SkupperNamespace = cli.GetNamespace()
+				secret, err := cli.ConnectorCreateFromFile(context.Background(), args[0], connectorCreateOpts)
+				if err != nil {
+					return fmt.Errorf("Failed to create connection: %w", err)
+				} else {
+					if siteConfig.Spec.IsEdge {
+						fmt.Printf("Skupper configured to connect to %s:%s (name=%s)\n",
+							secret.ObjectMeta.Annotations["edge-host"],
+							secret.ObjectMeta.Annotations["edge-port"],
+							secret.ObjectMeta.Name)
+					} else {
+						fmt.Printf("Skupper configured to connect to %s:%s (name=%s)\n",
+							secret.ObjectMeta.Annotations["inter-router-host"],
+							secret.ObjectMeta.Annotations["inter-router-port"],
+							secret.ObjectMeta.Name)
+					}
+				}
```

Comment:
This code works just fine, but just to reduce indentation level, you could have the code in the `else` block just below the `if`. 

The same applies to other changes in this PR.